android/src/main/java/com/reactnativekeyboardcontroller/views/EdgeToEdgeReactViewGroup.kt

```diff
@@ -200,4 +202,20 @@ class EdgeToEdgeReactViewGroup(private val reactContext: ThemedReactContext) : R
     }
   }
   // endregion
+
+  // region external methods
+  fun forceStatusBarTranslucent(isStatusBarTranslucent: Boolean) {
+    UiThreadUtil.runOnUiThread {
+      if (active) {
```

Comment:
I think we need to change here only if `this.isStatusBarTranslucent != isStatusBarTranslucent`?